### PR TITLE
Feat: multiple hosts in ingress

### PR DESCRIPTION
**Pull request by mikolajsobolewski**

### CODE REVIEW

**Comment by jlehtimaki** [comment]
Why is this needed?
Why not create a separate ingress for different purposes?

File: charts/landing/Chart.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.3
+version: 0.1.4
```
Comment (jlehtimaki) [inline charts/landing/Chart.yaml:19]: Not a bug fix but a feature. Please follow semver conventions.

```suggestion
version: 0.2.0
```

**Comment by mikolajsobolewski** [comment]
we have two urls for one service: wardenprotocol.org and genesis.wardenprotocol.org. and we can use simple enought description for ingress. Like this
```
ingress:
  enabled: true
  className: "kong"
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt
  hosts:
    - name: "wardenprotocol.org"
      paths:
        - port: 3000
    - name: "genesis.wardenprotocol.org"
      paths:
        - port: 3000
  tls:
    - hosts:
        - "wardenprotocol.org"
        - "genesis.wardenprotocol.org"
```

File: charts/landing/Chart.yaml (hunk 1 above)
Comment (mikolajsobolewski) [reply charts/landing/Chart.yaml:19]: we have two urls for one service: wardenprotocol.org and genesis.wardenprotocol.org. and we can use simple enought description for ingress. Like this
```
ingress:
  enabled: true
  className: "kong"
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt
  hosts:
    - name: "wardenprotocol.org"
      paths:
        - port: 3000
    - name: "genesis.wardenprotocol.org"
      paths:
        - port: 3000
  tls:
    - hosts:
        - "wardenprotocol.org"
        - "genesis.wardenprotocol.org
```

**Comment by mikolajsobolewski** [comment]
> Why is this needed? Why not create a separate ingress for different purposes?

It needs for landing we have two urls wardenprotocol.org and genesis.wardenprotocol.org. When we need to create that we can use a simple enough syntax:
```
ingress:
  enabled: true
  className: "kong"
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt
  hosts:
    - name: "wardenprotocol.org"
      paths:
        - port: 3000
    - name: "genesis.wardenprotocol.org"
      paths:
        - port: 3000
  tls:
    - hosts:
        - "wardenprotocol.org"
        - "genesis.wardenprotocol.org"
```
I think syntax with usage more than one ingress is more heavy.

**Review by jlehtimaki — APPROVED** [review]

**Pull request merged**